libraries/AP_AdvancedFailsafe/AP_AdvancedFailsafe.cpp

```diff
@@ -279,7 +279,7 @@ AP_AdvancedFailsafe::check(uint32_t last_valid_rc_ms)
             gcs().send_text(MAV_SEVERITY_DEBUG, "AFS State: AFS_AUTO, GCS now OK");
             // we only return to the mission if we have not exceeded AFS_MAX_COM_LOSS
             if (_saved_wp != 0 && 
-                (_max_comms_loss <= 0 || 
+                (_max_comms_loss == 0 || 
                  _comms_loss_count <= _max_comms_loss)) {
```

Comment:
```suggestion
                 _comms_loss_count <= labs(_max_comms_loss))) {
```
I think we need abs?

Reply:
Hi @IamPete1 ,

I decided not to mess with the old code. I added AFS_OPIONS as you suggested instead.